integration/test/basic/main_test.go

```diff
@@ -8,36 +8,27 @@ import (
 	"os"
 	"testing"
 
-	"github.com/giantswarm/appcatalog"
-	e2esetup "github.com/giantswarm/e2esetup/v2/chart"
-	"github.com/giantswarm/e2esetup/v2/chart/env"
-	"github.com/giantswarm/e2etests/v2/basicapp"
-	"github.com/giantswarm/helmclient/v2/pkg/helmclient"
-	"github.com/giantswarm/k8sclient/v4/pkg/k8sclient"
+	"github.com/giantswarm/apptest"
 	"github.com/giantswarm/micrologger"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
+	"github.com/giantswarm/coredns-app/integration/env"
 	"github.com/giantswarm/coredns-app/integration/templates"
 )
 
+
 const (
-	app            = "coredns"
+	app            = "coredns-test"
 	appName        = "coredns-app"
-	catalogURL     = "https://giantswarm.github.io/default-catalog"
-	testCatalogURL = "https://giantswarm.github.io/default-test-catalog"
+	catalogName    = "default-test"
+	testNamespace      = "default"
```

Comment:
This seems like the `fmt` is not correct - shouldn't the equals signs be aligned?

I assume `default` namespace is because the kind coreDNS is running in kube-system?

Reply:
I ran go fmt on main_test.go. Thanks for pointing that out :)

> I assume default namespace is because the kind coreDNS is running in kube-system?

That is correct.